index.html

```diff
@@ -5529,16 +5537,20 @@ <h3>Thing Description JSON-LD Context</h3>
     "id": "urn:dev:ops:32473-WoTLamp-1234",
     "title": "MyLampThing",
     "@type": "saref:LightSwitch",
+    "saref:hasState": {
+        "@id": "urn:dev:ops:32473-WoTLamp-1234/state",
+        "@type": "saref:OnOffState"
```

Comment:
The type OnOffState in SAREF has blanks. 'saref:On off state'
If we have a name without blanks, who is doing the mapping?

Reply:
I assume you refer to the HTML documentation of SAREF showing 'saref:On off state'. This UI is not the best but the full IRI is shown right above the title as `https://w3id.org/saref#OnOffState`:

![image](https://user-images.githubusercontent.com/6052970/57884743-a9d7df80-7829-11e9-8723-436256e6ccbe.png)

The title, with spaces, is in fact made of the human-readable label of the term, appended to the vocabulary prefix. It is however just a rendered view and not the canonical IRI for the term. I understand it is confusing.